media-service/settings.gradle

```diff
@@ -0,0 +1,10 @@
+rootProject.name = 'tabletop-games-shop'
+include 'core'
+include 'abstract-user-service'
+include 'user-service'
+include 'notification-service'
+include 'abstract-notification-service'
+include 'common'
+include 'abstract-media-service'
+include 'media-service'
+
```

Comment:
Why do you need the settings.gradle in this module? settings.gradle should be created only in base module (at the same level as base bulid.gradle). So, consider extending existing settings.gradle with include 'media-service' instead of creating it here.